### DISCUSSION TITLE
How to configure eslint in a static sandbox?

### DISCUSSION THREAD

**Discussion by PullJosh** [codesandbox/codesandbox-client#4503] [General] [answered]
I'd like to disable some eslint rules for all files, but I don't know how. Adding `.eslintrc.json` doesn't seem to work. Any advice?

**Comment by garethx** [comment, marked as answer]
You can set it per file like `/* eslint-disable no-undef, no-unused-vars */`, but I don't know of a supported global way.

**Reply by PullJosh** [reply]
That's a little disappointing, but I appreciate the info. 👍

**Comment by allforabit** [comment]
This doesn't seem to work for create react app (with typescript apps). It makes it very hard to distinguish between what's actually an error and something that I just haven't hooked into the code yet.

**Reply by SaraVieira** [reply]
Hey

I said in the previous comment in the case of TS another comment must be added since when you use TS we activate the TS eslint errors as well.

Should work with

```
/* eslint-disable no-undef, @typescript-eslint/no-unused-vars, no-unused-vars */
```

Example
https://codesandbox.io/s/disable-unused-vars-example-forked-fz6ki?file=/src/App.tsx:0-80

**Reply by allforabit** [reply]
Yes sorry for cross posting. Thanks again!
